### PR TITLE
Enhancement #13 remove_stopwords feature added

### PR DESCRIPTION
**Pull request by HammadRafique29**
I have gone through all the code, found some bugs, and fixed them. Now **find_similar_api** with `remove_stops_words` is working perfectly.

Here is the following things to be done for proper execution.

- `py manage.py` runserver throws an error **_('inspect' has no attribute 'getargspec'.)_**
- Above error is because **getargspec** is removed from **pymorphy2**. I have Opened an issue, with pull request on pymorphy2 [repo 
](https://github.com/pymorphy2/pymorphy2/pull/167)
- **Find_Similar** using latest version with updated **remove_stopwords** functionality. But **find_similar_demo** `requirements.txt` contains the outdated requirement
- Added some code in files serializers.py for passing remove_stopwords argument.
- Increases the effeciency of stop_words using nltk.stopwords
- After all of these above steps, **find_similar_demo** is running perfectly. In following output by passing `remove_stopwords=true` string **"is"** in text is removed

**For adding** `nltk.stopwords` **functionality, i need to pull request to Find-Similar repo. But its not avaiable for fork**

### CODE REVIEW

File: features/serializers.py [hunk 1]
```diff
@@ -13,6 +13,7 @@ class FindSimilarSerializer(serializers.Serializer):
     language = serializers.CharField(required=False)
     count = serializers.IntegerField(default=5)
     dictionary = serializers.DictField(required=False)
+    remove_stopwords = serializers.BooleanField(required=False)
```
Comment (quillcraftsman) [inline features/serializers.py:16]: I haven't seen that GitHub action with tests was run. Was all test passed and coverage wasn't changed?

File: requirements.txt [hunk 2]
```diff
@@ -1,7 +1,8 @@
 Django==4.2.4
-find-similar==1.0.1
+find-similar
```
Comment (quillcraftsman) [inline requirements.txt:2]: API can use different versions of library (not only last). In this way we can control API version independent of `find-similar` library version. Now we can use 1.2.0 version. Can you fix this concrete version in `requirements.txt`?

File: requirements.txt [hunk 3]
```diff
@@ -1,7 +1,8 @@
...
 djangorestframework==3.14.0
 markdown2==2.4.10
 django-filter==23.2
 django-cors-headers==4.2.0
-drf-yasg==1.21.7
\ No newline at end of file
+drf-yasg==1.21.7
+nltk
```
Comment (quillcraftsman) [inline requirements.txt:8]: `nltk` installs with `find-similar`. Don't understand why should we duplicate it here?

**Review by quillcraftsman — COMMENTED** [review]
This is useful working patch. Some comments has been added

File: requirements.txt (hunk 3 above)
Comment (HammadRafique29) [reply requirements.txt:8]: Stop_word means the words you want to skip like 
'is', "a", "the" etc

Lets take an example:

Comparable_txt = "Ali"
Text = ["Ali Ahmad", "is Ali"]
remove_stopwords = true

Here in above example, there is word "is" in text. We set remove_stopwords=true, this means the words that is useless will be removed. Hence the answer should be

Ali
1.0
Ali Ahmad
0.7

Problem:
When i run this example on find_simple with remove_stopwords it returned

Ali Ahmad
0.7
is Ali
0.7

See, useless word with no meaning which have to be removed still there. So i add the NLTK library to get prebuilt nltk.stop_words to handle this scenerio, and it runs successfully.

File: requirements.txt (hunk 2 above)
Comment (HammadRafique29) [reply requirements.txt:2]: Ok, i will set it to find-similar==1.2.0

File: requirements.txt (hunk 3 above)
Comment (HammadRafique29) [reply requirements.txt]: As you said nltk is being used with find-simple, so i just remove this duplicate

File: features/serializers.py (hunk 1 above)
Comment (HammadRafique29) [reply features/serializers.py:16]: image

**Pull request merged**